src/renderers/noop/ReactNoop.js

```diff
@@ -210,6 +211,10 @@ var ReactNoop = {
     NoopRenderer.performWithPriority(AnimationPriority, fn);
   },
 
+  setDefaultPriority(priorityLevel: PriorityLevel) {
+    NoopRenderer.setDefaultPriority(priorityLevel);
+  },
```

Comment:
Can we just make this low pri by default in ReactNoop and synchronous in ReactDOMFiber? So we don't have to expose this as an API?